.github/workflows/main.yml

```diff
@@ -98,6 +98,7 @@ jobs:
           level: warning
           # To be customized to cover remaining Python scripts:
           glob_pattern: "**/*.py"
+        continue-on-error: true
```

Comment:
Now it won't block when there are actual errors, but at least will still make the annotations visible